MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: Customizations in MAUI Calendar widget | Syncfusion
+description: Learn here all about Customizations features of Syncfusion .NET MAUI Calendar (SfCalendar) widget and more. 
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Customizations in MAUI Calendar (SfCalendar)
+Each cell of the Month, Year, Decade and Century can be customized in MAUI Calendar (SfCalendar).
+
+## Month cell customization
+You can customize the calendar month view cell by using the `MonthView` property of `SfCalendar`.
+
+*    **Month dates** – You can customize the month date's text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `MonthView`.
```

Comment:
You can customize the month dates' text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `MonthView.`

Reply:
Content changed